blocks.md

```diff
@@ -115,6 +115,8 @@ Key | Values | Required | Default
 `driver` | One of `"sysfs"` or `"upower"`. | No | `"sysfs"`
 `interval` | Update interval, in seconds. Only relevant for `driver = "sysfs"`. | No | `10`
 `format` | A format string. See below for available placeholders. | No | `"{percentage}%"`
+`allow_missing_battery` | Don't display errors when the battery cannot be found. Only works with the `sysfs` driver. | No | `false`
```

Comment:
Perhaps we could shorten this to `allow_missing`?

Reply:
Yep, I agree. The battery is redundant in this context